openapi.yaml

```
@@ -746,6 +746,69 @@ paths:
               schema:
                 $ref: "#/components/schemas/Error"
 
+  /statistics/cardano-database/immutable-files-restored:
+    post:
+      summary: Records immutable files restored event
+      description: Records immutable files restored event
```

Could you do the same modification in all the other messages below:
```suggestion
      summary: Records Cardano database immutable files restored event
      description: Records Cardano database immutable files restored event
```